src/arnoldi.jl

```diff
@@ -51,6 +52,12 @@ function Base.show(io::IO, Ks::KrylovSubspace)
     println(IOContext(io, :limit => true), getH(Ks))
 end
 
+# Helper functions that returns the real part if that is what is
+# required (for Hermitian matrices), otherwise returns the value
+# as-is.
+coeff(::Type{T},α::T) where {T} = α
+coeff(::Type{U},α::T) where {U<:Real,T<:Complex} = real(α)
```

Comment:
This could just be `real`, since `real(x::AbstractFloat) == x`

Reply:
But it would not work for the case when `T<:Complex`, which is the usual case for complex Arnoldi.